src/content/docs/plugins/submission.md

```diff
@@ -1,4 +1,115 @@
 ---
-title: Plugin Submission
-description: How to submit your plugin to the official Vencord repository
+title: Plugin Submission & Updates
+description: Vencord Plugin submission and updates guide
 ---
+
+This guide describes steps on how to release a new plugin to Vencord and to make changes to the already existing plugin. 
+
+## Initial Plugin Submission 
+
+### Pre-submission Checklist 
+
+- The folder in which stores your plugins should have camelCase name (for instance, `myAwesomePlugin`). 
+- `README.md` is in your plugin folder with a description, usage instructions, screenshots(if any) 
+- The coding of your plugin adheres to Vencord’s coding standards 
+- This is the best because probably, you have engaged in rigorous testing of the plugin. 
+
+### Submission Steps 
+
+1. **Fork the Repository**: Go to the specific repository on [https://github.com/Vendicated/Vencord](https://github.com/Vendicated/Vencord) page and select “Fork”. 
+
+2. **Clone Your Fork** 
```

Comment:
This is too oversimplified; There are a decent amount of people who have decided to install userplugins before getting started. Change this to branch into a "if you've installed userplugins set the remote and upstream branches" and an "if not clone the fork".

Reply:
Fixed :D